build/ansible/roles/lvm-init/tasks/main.yml

```diff
@@ -1,4 +1,14 @@
 ---
+- name: Remove device-mapper package without dependencies
+  command:
+    cmd: rpm -e --nodeps device-mapper
```

Comment:
Any insights as to why we are removing the device-mapper package?

Reply:
In pmm/build/ansible/roles/lvm-init/tasks/main.yml, we are installing lvm2 on CentOS 7, but it is failing due to a dependency issue with the device-mapper package. The version installed on the new available AMI for CentOS 7 is incompatible with lvm2, and we cannot remove the device-mapper package because it attempts to remove systemd, which is protected, resulting in the error:

```Error: Trying to remove "systemd", which is protected```

Since CentOS 7 is EOL and not supported, we are resolving this issue by forcefully removing the device-mapper package without its dependencies.